diff --git utils.bzl utils.bzl
index 5509f97..c18f65e 100644
--- utils.bzl
+++ utils.bzl
@@ -38,12 +38,7 @@ def get_arch_windows(rctx):
     fail("Unsupported architecture: {}".format(arch))
 
 def get_arch_mac(rctx):
-    arch = rctx.execute(["uname", "-m"]).stdout.strip("\n")
-    if arch in ["x86_64", "amd64"]:
-        return "x86_64"
-    elif arch in ["arm64"]:
-        return "arm64"
-    fail("Unsupported architecture: {}".format(arch))
+    return "x86_64"
 
 def get_arch_linux(rctx):
     arch = rctx.execute(["uname", "-m"]).stdout.strip("\n")
